site/content/user-guide/http-adapter.md

```diff
@@ -28,7 +28,12 @@ There is a subtle difference between the *device identifier* (*device-id*) and t
   * 202 (Accepted): The telemetry data has been accepted for processing. Note that this does not *guarantee* successful delivery to potential consumers.
   * 400 (Bad Request): The request cannot be processed because the content type header is missing or the request body is empty.
   * 401 (Unauthorized): The request cannot be processed because the request does not contain valid credentials.
-  * 403 (Forbidden): The request cannot be processed because the device's registration status cannot be asserted, i.e. the given device either does not belong to the given tenant or is disabled.
+  * 403 (Forbidden): The request cannot be processed because the device's registration status cannot be asserted. Possible reasons for this might be:
+
+      The given tenant is not allowed to use this protocol adapter.  
```

Comment:
this needs to be formatted as a sublist:
```
  * 403 (Forbidden): The request cannot be processed because the device's registration status cannot be asserted. Possible reasons for this might be:
        * The given tenant ...
        * The given device ...
```
That is *eight* spaces at the beginning of the line. The list of possible reasons should then be indented correctly at the third level.